suites/reef/nvmeof/tier-1_nvmeof_librbd_vs_nvmeof_16k_block_size.yaml

```diff
@@ -0,0 +1,173 @@
+# Basic IO Perf comparison test LibRBD vs NVMeoF protocols
+# cluster configuration: conf/quincy/nvmeof/ceph_nvmeof_sanity.yaml
+
+tests:
+
+# Set up the cluster
+  - test:
+      abort-on-fail: true
+      module: install_prereq.py
+      name: install ceph pre-requisites
+  - test:
+      abort-on-fail: true
+      config:
+        verify_cluster_health: true
+        steps:
+          - config:
+              command: bootstrap
+              service: cephadm
+              args:
+                mon-ip: node1
+                registry-url: registry.redhat.io
+                allow-fqdn-hostname: true
+          - config:
+              command: add_hosts
+              service: host
+              args:
+                attach_ip_address: true
+                labels: apply-all-labels
+          - config:
+              command: apply
+              service: mgr
+              args:
+                placement:
+                  label: mgr
+          - config:
+              command: apply
+              service: mon
+              args:
+                placement:
+                  label: mon
+          - config:
+              command: apply
+              service: osd
+              args:
+                all-available-devices: true
+      desc: RHCS cluster deployment using cephadm
+      destroy-clster: false
+      module: test_cephadm.py
+      name: deploy cluster
+
+#  Test cases to be executed
+  - test:
+      abort-on-fail: true
+      config:
+        command: add
+        id: client.1
+        nodes:
+          - node6
+          - node7
+        install_packages:
+          - ceph-common
+          - fio
+        copy_admin_keyring: true
+      desc: Setup client on NVMEoF gateway
+      destroy-cluster: false
+      module: test_client.py
+      name: configure Ceph client for NVMe tests
+      polarion-id: CEPH-83573758
+
+  # Test IO Perf
+  - test:
+      name: libRBD VS NVMeoF - Read and Write IOtype
+      description: Write and Read IO comparison on 10G image with 16kb-block-size
+      module: test_io_perf.py
+      config:
+        iterations: 1               # number of iterations to find out average
+        io_profiles:
+          - FIO_WRITE_BS_4k_IODepth8_LIBAIO
+          - FIO_READ_BS_4k_IODepth8_LIBAIO
+        io_overrides:
```

Comment:
@sunilkumarn417 considering the combinations we had/ going to have on io_type, block_size and any unqiue io args for fio, IMO we can pass all of them as overrides, say like here we can pass io_type as override. 

This will reduce number of entries at https://github.com/red-hat-storage/cephci/blob/master/utility/io/fio_profiles.py and have only 3 entries - read, writes and readwrite(replace every parameter in each entry through overrides) or 2 entries if we combine for read and write. 

This also  provides flexibility to specify them at test suite itself. Thoughts?

Reply:
@rahullepakshi , yes in a way its makes sense. lets discuss more and we could take this in next PR as an improvement.